.github/workflows/spark-integration-test.yml

```diff
@@ -92,9 +92,13 @@ jobs:
         run: |
           if [ "${{ matrix.scala-version }}" == "2.12" ];then 
             ./gradlew -PskipTests -PtestMode=${{ matrix.test-mode }} -PjdkVersion=${{ matrix.java-version }} -PscalaVersion=${{ matrix.scala-version }} -PskipDockerTests=false :spark-connector:spark-3.3:test --tests "org.apache.gravitino.spark.connector.integration.test.**"
+            ./gradlew -PskipTests -PtestMode=${{ matrix.test-mode }} -PjdkVersion=${{ matrix.java-version }} -PscalaVersion=${{ matrix.scala-version }} -PskipDockerTests=false :spark-connector:spark-3.4:test --tests "org.apache.gravitino.spark.connector.integration.test.**"
+            ./gradlew -PskipTests -PtestMode=${{ matrix.test-mode }} -PjdkVersion=${{ matrix.java-version }} -PscalaVersion=${{ matrix.scala-version }} -PskipDockerTests=false :spark-connector:spark-3.5:test --tests "org.apache.gravitino.spark.connector.integration.test.**"
+          fi
+          if [ "${{ matrix.scala-version }}" == "2.13" ];then 
+            ./gradlew -PtestMode=${{ matrix.test-mode }} -PjdkVersion=${{ matrix.java-version }} -PscalaVersion=${{ matrix.scala-version }} :spark-connector:spark-3.4:test
```

Comment:
I think we must provide a way to test for compatibility with scala2.13, and if adding scala2.13 will make the pipelines too slow, we can skip docker tests. WDYT? @FANNG1

Reply:
How about add spark connector build and UT with scala2.13  in `.github/workflows/build.yml`?

Reply:
ok for me.

Reply:
updated.